lib/css/components/_stacks-usercards.less

```diff
@@ -0,0 +1,91 @@
+//
+//  STACK OVERFLOW
+//  USER CARDS
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  ============================================================================
+//  $  BASE
+//  ----------------------------------------------------------------------------
+
+
+.s-usercard {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-column-gap: 10px;
+
+    &--time {
```

Comment:
These are nice and clever, but they make it impossible to search for `.s-usercard--time` in the codebase. Let's be verbose here.